lib/spreedly/transactions/purchase.rb

```diff
@@ -1,7 +1,6 @@
 module Spreedly
-
   class Purchase < AuthPurchase
-
+    field :checkout_url, :checkout_form, :redirect_url, :callback_url, \
+          :required_action, :challenge_form, :challenge_url, :device_fingerprint_form
```

Comment:
I presume that authorization transactions would be included in both the 3DS1 and 3DS2 flows. Should these new attributes be added to the `AuthPurchase` base instead?

Reply:
Yep, nice catch!